indexer/src/lib.rs

```diff
@@ -91,7 +91,10 @@ where
         for index in &mut self.indexes {
             writers.push(block_in_place(|| index.writer())?);
         }
-        let consumer = self.bus.subscribe("indexer", &[self.stored_topic]).await?;
+        let consumer = self
+            .bus
+            .subscribe(&format!("indexer-{}", self.storage.name()), &[self.stored_topic])
```

Comment:
I can appreciate that. I also don't know the answer.

Reply:
@bobmcwhirter I reverted this change, opting instead to configure the bombastic bucket to send its change events to both the bombastic and vexination queues. I think this will work better for both kafka and SQS.

This will definitely require a change to the terraform scripts, but I'm not sure how to do that yet, much less confirm it works. :man_shrugging: